index.bs

```diff
@@ -481,21 +487,21 @@ accept the incoming connection:
    running these steps and act as if the requested service is not
    available.
 
-2. [=Get a session ID for a WebSocket resource=] with |resource name|
+1. [=Get a session ID for a WebSocket resource=] with |resource name|
```

Comment:
If we can get https://github.com/w3c/webdriver-bidi/pull/138 merged first, that'd avoid a bunch of these changes in this PR.